Inspicio/wwwroot/css/site.css

```diff
@@ -1,16 +1,21 @@
 ﻿body {
-    padding-top: 50px;
-    padding-bottom: 20px;
+    padding-top: 0px;
+    padding-bottom: 0px;
+    font-family:'Century Gothic';
+    padding-left: 0px; 
 }
 
 hr {
     border-color: #e6e6e6;
 }
+nav{
+    border-radius:0!important;
```

Comment:
The "!important" is currently preventing the navigation bar from being rounded. Ideally the nav bar will be fixed in the future, so might be better to change it later on once the other pages are sorted.